kansha/card/comp.py

```diff
@@ -83,52 +70,9 @@ def board(self):
     def reload(self, data=None):
```

Comment:
Why such a `reload()` method. And, if needed, it's called `refresh()` in the board component